resharper/resharper-unity/src/Application/UI/Options/VisualStudio/ReSharperOptionsPage.cs

```diff
@@ -0,0 +1,51 @@
+using JetBrains.Annotations;
+using JetBrains.Application.Environment;
+using JetBrains.Application.Environment.Helpers;
+using JetBrains.Application.UI.Options;
+using JetBrains.DataFlow;
+using JetBrains.ReSharper.Feature.Services.OptionPages.CodeEditing;
+using JetBrains.ReSharper.Plugins.Unity.Resources;
+using JetBrains.ReSharper.Plugins.Unity.Settings;
+
+namespace JetBrains.ReSharper.Plugins.Unity.Application.UI.Options.VisualStudio
+{
+    // TODO: Merge with Rider's page?
+    // That would leave us with #ifdef soup...
+    [OptionsPage(PID, Name, typeof(LogoThemedIcons.UnityLogo), ParentId = CodeEditingPage.PID)]
```

Comment:
1. We can use be-controls in 191
2. VS options will be base class, Rider will extend them - this means no #ifdef